contrib/mac/app/.gitignore

```diff
@@ -1,3 +1,4 @@
 julia/
 dmg/
 *.dmg
+notarize-*.xml
```

Comment:
PSA: unless a git ignore pattern starts with a `/` it applies anywhere beneath the directory containing the git ignore file. If you want to ignore just a specific top-level file with a given name, start the pattern with a `/`.